docs/examples/control_ish_ufschem-example.yaml

```diff
@@ -0,0 +1,294 @@
+# General Description:
+# - Any key that is specific for a plot type will begin with `ts` for timeseries, `ty` for taylor.
+# - Some keys/groups are optional.
+# - For now, all plots except time series average over the analysis window.
+# - Setting axis values
+#   - If set_axis = True in data_proc section of each plot_grp,
+#     the yaxis for the plot will be set based on the values
+#     specified in the obs section for each variable.
+#   - If set_axis is set to False, then defaults will be used.
+#   - 'vmin_plot' and 'vmax_plot' are needed for
+#     'timeseries', 'spatial_overlay', and 'boxplot'.
+#   - 'vdiff_plot' is needed for 'spatial_bias' plots
+#   - 'ty_scale' is needed for 'taylor' plots.
+#   - 'nlevels' or the number of levels used in the contour plot can also optionally be provided for spatial_overlay plot.
+#   - If set_axis = True and the proper limits are not provided in the obs section,
+#     a warning will print, and the plot will be created using the default limits.
+
+analysis:
+  start_time: "2017-07-01-00:00:00" # UTC
+  end_time: "2017-07-03-00:00:00" # UTC
+  output_dir: ./output/ish_ufschem  # relative to the program using this control file
+  #   Currently, the directory must exist or plot saving will error and fail.
+  debug: True
+
+model:
```

Comment:
In order to get the contour plots to look nice without the white line down the middle. You can add  is_global: True in the model description like this one:  https://github.com/NCAR/MELODIES-MONET/blob/main/examples/yaml/control_ufsaqm_mopitt.yaml